Source/Core/DolphinQt/MainWindow.h

```diff
@@ -11,11 +11,12 @@
 #include <optional>
 #include <string>
 
+#include "Core/Boot/Boot.h"
```

Comment:
this appears required in this header because the type is used as a defaulted argument in some prototypes defined here.